src/modules/billing/mappers/contact.js

```diff
@@ -3,6 +3,14 @@
 const { isEmpty } = require('lodash');
 const Contact = require('../../../lib/models/contact-v2');
 
+const getInitials = contact => {
```

Comment:
I think it would give more flexibility to add a `middleInitials` property to the Contact model above rather than map everything to the existing `initials` property